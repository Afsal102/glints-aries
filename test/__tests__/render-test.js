/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

jest.mock('uikit-manifest');
jest.mock('react-dom', () => ({
  findDOMNode: () => {},
}));
import UikitManifest from 'uikit-manifest';

import IntroStories from '../../stories/IntroStory';

// import UikitProvider from '../../src/Provider';

test('Render Intor story correctly', () => {
  const mockUikitManifest = UikitManifest.getManifest();
  const tree = renderer
    .create(
      <IntroStories />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
