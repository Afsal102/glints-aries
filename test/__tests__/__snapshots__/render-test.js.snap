// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Intor story correctly 1`] = `
<div
  className="doc-mainbar"
>
  <h1>
    Installation
  </h1>
  <p>
    npm: 
    <code>
      npm install -S [package name]
    </code>
  </p>
</div>
`;
